KaijuKiller/Assets/Scripts/PlayerMove.cs

```diff
@@ -58,4 +74,14 @@ IEnumerator stopLaneChange(int i)
         bounds += i;
         isLaneChanging = false;
     }
+
+    IEnumerator stopJump()
+    {
+       //lane change bug
+        yield return new WaitForSeconds(0.5f);
+        rb.velocity = new Vector3(0, -5, moveSpeed);
+        yield return new WaitForSeconds(0.5f);
+        rb.velocity = new Vector3(0, 0, moveSpeed);
+        isJumping = false;
+    }
```

Comment:
Using a coroutine to stop a jump is concerning. Because your object has a rigidbody, why not let the physics system take care of it.

I don't exactly know what your game is so this may be necessary. It just put up a red flag for me